doc/ref/chap-osh-assign.md

```diff
@@ -239,20 +238,21 @@ item in order.  An item in the initializer list must be in the forms
 `[<key>]=<value>` or `[<key>]=<value>`.  The element to be modified is
 specified by `<key>`.
 
-    declare -A a                # This creates an empty BashAssoc (OSH)
-    declare -A a=()             # This creates an empty BashAssoc
-    declare -A a=([a]=1 [b]=2)  # This creates a BashAssoc with two elements
+    declare -A a                 # This creates an empty BashAssoc (OSH)
+    declare -A a=()              # This creates an empty BashAssoc
+    declare -A a=([a]=1 [b]=2)   # This creates a BashAssoc with two elements
 
-    declare -A a=(1 2)          # This is an error (OSH)
+    declare -A a=(1 2)           # This is an error (OSH)
 
     k=10
-    declare -A a=([k]=v)        # This creates a BashAssoc with one element,
-                                # (['k']=1).  Unlike BashArray, "k" is not
-                                # processed by arithmetic expansion.
-    declare -a a+=([a]=3 [b]=4) # This adds two elements to the existing array.
-                                # The result is ([a]=3 [b]=4 [k]=v)
-    declare -a a+=([k]=5)       # This overwrites an element in the existing
-                                # array. The result is ([a]=3 [b]=4 [k]=5).
+    declare -A a=([k]=v)         # This creates a BashAssoc with one element,
+                                 # (['k']=1).  Unlike BashArray, "k" is not
+                                 # processed by arithmetic expansion.
+    declare -a a+=([a]=3 [b]=4)  # This adds two elements to the original
```

Comment:
should this just be `a+=()` without `declare -a`?

If we are going to include the flag, it should be -A right?

Reply:
Ah, you are right. I just created these examples by rewriting those for BashArray, and I forgot to rewrite the flags.

I decided to remove `declare -a` / `declare -A`. I fixed it in commit 19d86b06e.